jazz_test-lambda/index.js

```diff
@@ -0,0 +1,122 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+Nodejs Template Project
+@author:
+@version: 1.0
+ **/
+
+const errorHandlerModule = require("./components/error-handler.js"); //Import the error codes module.
+const responseObj = require("./components/response.js"); //Import the response module.
+const configObj = require("./components/config.js"); //Import the environment data.
+const logger = require("./components/logger.js"); //Import the logging module.
+const utils = require("./components/utils.js");
+const aws = require('aws-sdk');
+const validateARN = utils.validateARN;
+const execStatus = utils.execStatus();
+
+var handler = (event, context, cb) => {
+
+  'use strict';
+  //Initializations
+  var errorHandler = errorHandlerModule();
+  var config = configObj(event);
+  logger.init(event, context);
+  var awsRegion;
+  try {
+    var testResponse = {
+      "execStatus": null,
+      "payload": null,
+    };
+    if (!event && !event.method && event.method !== 'POST') {
+      return cb(JSON.stringify(errorHandler.throwNotFoundError("Method not found")));
+    }
+    if (!event.body) {
+      return cb(JSON.stringify(errorHandler.throwInputValidationError("Request payload cannot be empty")));
+    }
+    if (!validateARN(event.body.functionARN)) {
+      return cb(JSON.stringify(errorHandler.throwInputValidationError("Function ARN is invalid")));
+    }
+    if (!event.body.inputJSON) {
+      return cb(JSON.stringify(errorHandler.throwInputValidationError("Input for function is not defined")));
+    }
+    var functionARN = event.body.functionARN;
+    var arnvalues = functionARN.split(":");
+    awsRegion = arnvalues[3]; //["arn","aws","lambda","us-east-1","000000""] spliting FunctionARN to get the aws-region
+    var inputJSON = event.body.inputJSON;
+
+    invokeLambda(functionARN, inputJSON, awsRegion).then((data) => {
+
+      if (data && data.StatusCode >= 200 && data.StatusCode < 299) {
+        testResponse.payload = data;
+        if (!data.FunctionError) {
+          //Function Executed Succesfully Without Error
+          testResponse.execStatus = execStatus.success;
+        } else {
+          if (data.FunctionError === "Handled") {
+            testResponse.execStatus = execStatus.handledError;
+          } else if (data.FunctionError === "Unhandled") {
+            // Function Execution Had Unhandled Error
+            testResponse.execStatus = execStatus.unhandledError;
+          }
+        }
+      } else {
+        // Function Falied |Cause Unknown|TEST FAILED
+        return cb(JSON.stringify(errorHandler.throwInternalServerError("Unknown internal error occurred when invoking " + functionARN)));
+      }
+      testResponse.payload = data;
+      return cb(null, responseObj(testResponse, event.body));
+    }).catch((err) => {
+      // Funtion Failed To Be Invoked |TEST FAILED
```

Comment:
log error. Also remove the comment here.

Reply:
comment removed ...  This error is getting logged in line no 105 | it would be a duplicate